src/Protocol/DFRN.php

```diff
@@ -2209,26 +2207,26 @@ private static function processDeletion(DOMXPath $xpath, DOMNode $deletion, arra
 		$condition = ['uri' => $uri, 'uid' => $importer['importer_uid']];
 		$item = Post::selectFirst(['id', 'parent', 'contact-id', 'uri-id', 'deleted', 'gravity'], $condition);
 		if (!DBA::isResult($item)) {
-			Logger::info("Item with uri " . $uri . " for user " . $importer['importer_uid'] . " wasn't found.");
+			Logger::info('Item with URI ' . $uri . ' for user ' . $importer['importer_uid'] . ' was not found.');
```

Comment:
So, the purpose of normalizing quotes to single quotes is to be able to spot the double-quote strings when they are actually needed like in this case where we want to use a quote in a string. We don't have to avoid using double-quotes at all cost, they just need to be used only when they are needed.